server/api_server/graphql_endpoint.py

```
@@ -41,10 +43,95 @@
+                offset=offset,
+            )
+            total = h.get_total_group_count(
+                devGUID=devGuid,
+                changedColumn=changedColumn,
+                changedBy=changedBy,
+            )
+            return DeviceHistoryResult(
+                history=_to_graphql_history(groups),
+                count=total,
+            )
+        except Exception as e:
```

Ruff (0.15.20)

[warning] Do not catch blind exception: `Exception`

(BLE001)